lib/itamae/resource/remote_directory.rb

```diff
@@ -56,10 +58,18 @@ def show_differences
 
       def action_create(options)
         if attributes.mode
-          run_specinfra(:change_file_mode, @temppath, attributes.mode)
+          if attributes.recursive_mode == :true
+            run_specinfra(:change_file_mode, @temppath, attributes.mode, :recursive => true)
```

Comment:
If `attributes.recursive_mode` is boolean, you can simply write this `run_specinfra(:change_file_mode, @temppath, attributes.mode, recursive: attributes.recursive_mode)`